src/main/scala/molmed/config/FileAndProgramResourceConfig.scala

```diff
@@ -263,6 +209,15 @@ trait FileAndProgramResourceConfig {
       if (this.snpEffPath == null)
         this.snpEffPath = getFileFromKey(programNameToPathsMap, Constants.SNP_EFF)
 
+      if (this.novoalignCS == null)
```

Comment:
Good suggestion. And the comment about bwaPath is very reasonable. I did that changed before I decided to introduce novoalignCS in the xml file. But, again you are right, it is a good idea to move them Colospace things to a trait because right now they cause troubles to 'upsala' config.